components/upload/interface.tsx

```diff
@@ -27,6 +27,7 @@ export interface UploadFile<T = any> {
   status?: UploadFileStatus;
   percent?: number;
   thumbUrl?: string;
+  crossOrigin?: '' | 'anonymous' | 'use-credentials';
```

Comment:
```suggestion
  crossOrigin?: React.ImgHTMLAttributes<HTMLImageElement>['crossOrigin'];
```

Reply:
@afc163 Thank you. I realized that your suggestion is better type definition. I've just commited your suggestion. 👍